packages/core/src/types.ts

```diff
@@ -185,10 +202,17 @@ export interface LottieViewProps {
    */
   useNativeLooping?: boolean;
 
+  // Web Props
+
   /**
-   * A callback function which will be called when the animation loops.
-   *
-   * @platform windows
+   * A boolean flag to enable hover interactions. Whether to play on mouse hover. Defaults to false.
+   * @platform web
    */
-  onAnimationLoop?: () => void;
+  hover?: boolean;
```

Comment:
Since this is a web-only prop, should we warn the user when the prop is used on native just like we do with `progress` on web?